examples/decoding/plot_ems_filtering.py

```diff
@@ -57,38 +60,71 @@
 reject = dict(grad=4000e-13, eog=150e-6)
 
 epochs = mne.Epochs(raw, events, event_ids, tmin, tmax, picks=picks,
-                    baseline=None, reject=reject)
-
-# Let's equalize the trial counts in each condition
-epochs.equalize_event_counts(epochs.event_id, copy=False)
-
-# compute surrogate time series
-surrogates, filters, conditions = compute_ems(epochs, ['AudL', 'VisL'])
-
-times = epochs.times * 1e3
+                    baseline=None, reject=reject, preload=True)
+epochs.drop_bad()
+epochs.pick_types(meg='grad')
+
+# Setup the data to use it a scikit-learn way:
+X = epochs.get_data()  # The MEG data
+y = epochs.events[:, 2]  # The conditions indices
+n_epochs, n_channels, n_times = X.shape
+
+# Initialize EMS transformer
+ems = EMS()
+
+# initialize the variables of interest
+X_transform = np.zeros((n_epochs, n_times))  # Data after EMS transformation
+filters = list()  # Spatial filters at each time point
+
+# In the original paper, the cross-validation is a leave-one-out. However,
+# we recommend using a Stratified KFold, as LOO is known to generate biases.
+for train, test in StratifiedKFold(y):
+    # In the original paper, the z-scoring is applied outside the CV.
+    # However, we recommend to apply this preprocessing inside the CV.
+    # Note that such scaling should be done separately for each channels if the
+    # data contains multiple channel types.
+    # Note that X_scaled does not have any unit anymore.
+    X_scaled = X / np.std(X[train])
```

Comment:
Not sure how intransparent this would be for new users, but wouldn't this be a prime case for StandardScaler? (And the function requires sklearn anyways)